apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -101,11 +101,10 @@ public abstract partial class BigtableClient
             Task.Run(() => CreateAsync(clientCreationSettings, appProfileId)).ResultWithUnwrappedExceptions();
 
         /// <summary>
-        /// Synchronously creates a <see cref="BigtableClient"/>, applying defaults for all unspecified settings,
-        /// and creating a channel connecting to the given endpoint with application default credentials where
-        /// necessary.
+        /// Synchronously creates a <see cref="BigtableClient"/>, applying default settings.
```

Comment:
There are still no default settings applied. All settings are used to create the clients, which are being specified by the caller. Maybe just update this to be:
```csharp
/// Synchronously creates a <see cref="BigtableClient"/> from a set of pre-existing clients.
```